backend/utils/influencerAnalytics.js

```diff
@@ -27,32 +27,38 @@ const trackingData = () => {
 //Instagram
 const InstagramData = async (instagramId) => {
   const options = {
-    method: "GET",
-    url: process.env.INSTA_ENDPOINT,
+    method: 'GET',
+    url: config.INSTA_ENDPOINT,
     params: {
-      username_or_id: instagramId,
-      include_about: "true",
-      nocors: "true",
+      url: `https://www.instagram.com/${instagramId}/`
```

Comment:
2 times url required? please remove either to constant or remove it if not required